src/http/client.cr

```diff
@@ -378,93 +378,91 @@ class HTTP::Client
         yield response
       end
     end
-  {% end %}
 
-  {% for http_method in %w(post put patch) %}
-    # Executes a {{http_method.id.upcase}} request with form data and returns a `Response`. The "Content-Type" header is set
+    # Executes a {{method.id.upcase}} request with form data and returns a `Response`. The "Content-Type" header is set
     # to "application/x-www-form-urlencoded".
```

Comment:
Suggestion to add ``to "application/x-www-form-urlencoded". Otherwise it is the same as using `#{{method.id}}(path, headers, body)` ``

Reply:
Maybe, but it's not relevant to this PR.